content/en/docs/tasks/observability/gateways/index.md

```diff
@@ -17,23 +17,9 @@ two basic access methods: secure (via HTTPS) and insecure (via HTTP). The secure
 recommended* for any production or sensitive environment. Insecure access is simpler to set up, but
 will not protect any credentials or data transmitted outside of your cluster.
 
-### Option 1: Secure access (HTTPS)
-
-A server certificate is required for secure access. Follow these steps to install and configure
-server certificates for a domain that you control.
+For both options, first follow these steps:
 
-You may use self-signed certificates instead. Visit our
-[Securing Gateways with HTTPS](/docs/tasks/traffic-management/ingress/secure-ingress/)
-for general information on using self-signed certificates to access in-cluster services.
-
-{{< warning >}}
-This option covers securing the transport layer *only*. You should also configure the telemetry
-addons to require authentication when exposing them externally.
-{{< /warning >}}
-
-1. [Install cert-manager](/docs/ops/integrations/certmanager/) to manage certificates automatically.
-
-1. [Install Istio](/docs/setup/install/istioctl) in your cluster.
+1. [Install Istio](/docs/setup/install/istioctl) in your cluster with your desired telemetry addons.
```

Comment:
would remove that second clause since the additional telemetry addons are in the next line
```suggestion
1. [Install Istio](/docs/setup/install/istioctl) in your cluster.
```